README.md

```diff
@@ -105,15 +105,122 @@ always in sync, and that one or the other may be slightly more up-to or out-of d
+- other rows in `morbidmap.txt` where it appears as a phenotype having an association with another gene.
+  - In all such cases seen thus far as of 2024/11/18, all of these are cancer cases, and the label ends with "somatic".
+  - This entry appears in the Phenotype-Gene Relationships table on the MIM's omim.org/entry page.
+- Row 2: There is a second row where the phenotype in the first row appears as a gene.
+  - For this row, there is no MIM in the phenotype field.
+  - This row does not appear in the Gene-Phenotype Relationships table on the MIM's omim.org/entry page.
+  - This row is self-referential. The label in the Phenotype field is one of the titles of the MIM in the Gene field.
+
+**Example case**:
+|Phenotype|Gene/Locus And Other Related Symbols|MIM Number|Cyto Location|
+|-|-|-|-|
+|Small cell cancer of the lung, somatic, 182280 (3)|RB1|614041|13q14.2|
+|Small-cell cancer of lung (2)|SCLC1|182280|3p23-p21|
+
+
+**All known cases**:
+There is a spreadsheet which collates all known cases as of 2024/11/18: [google sheet](
+https://docs.google.com/spreadsheets/d/1hKSp2dyKye6y_20NK2HwLsaKNzWfGCMJMP52lKrkHtU/). The MIMs of the known cases are: `159595`, `182280`, `607107`, and `615830`.
+
+## Under the hood: Design decisions, etc.
+### Gene-Disease pipeline
+This pipeline involves the processing of `morbidmap.txt` to create ontological representations of Gene --> Disease and 
+Disease --> Gene associations.
+
+#### Example input/output
+##### Input: `morbidmap.txt`
+| Phenotype                        | Gene/Locus And Other Related Symbols | MIM Number | Cyto Location |
+|----------------------------------|--------------------------------------|------------|---------------|
+| Prune belly syndrome, 100100 (3) | CHRM3, PBS, EGBRS                    | 118494     | 1q43          |
+
+`OMIM:100100` (Prune belly syndrome) is the Phenotype ("Disease"), and `OMIM:118494` (CHRM3) is the associated Gene. 
+They are related via mapping key `(3)` (explained below).
+
+##### Output: `omim.ttl`
+```ttl
+OMIM:100100 a owl:Class ;
+    rdfs:label "prune belly syndrome" ;
+    rdfs:subClassOf _:N2fd22c9bb2f04630b81414cff9514660 ;
+    biolink:category biolink:Disease .
+    
+_:N2fd22c9bb2f04630b81414cff9514660 a owl:Restriction ;
+    owl:onProperty RO:0004003 ;
+    owl:someValuesFrom OMIM:118494 .
+```
+
+The association is represented as an `rdfs:subClassOf` `owl:Restriction`, where mapping key `(3)` is represented as 
+`RO:0004003`.
+
+#### OMIM MorbidMap mapping keys & Relationship Ontology predicates
+In order to add these associations to an OWL ontology, we must use an appropriate predicate. Below are the 4 OMIM 
+`morbidmap.txt` mapping keys and [their definitions](https://omim.org/help/faq#1_6), alongside the RO predicates we've 
+chosen to represent them.
+
+Note that the directionality of these associations / predicates is in the Disease->Gene direction:
+(Disease MIM) --(Mapping key / RO predicate)--> (Gene MIM)
```

Comment:
- [ ] Flip this. This section should say that these predicates are used in the Gene->Disease direction.